google-cloud-logging/lib/google/cloud/logging/entry.rb

```diff
@@ -473,7 +473,7 @@ def labels_grpc
           return {} if labels.nil?
           # Coerce symbols to strings
           Hash[labels.map do |k, v|
-            v = String(v) if v.is_a? Symbol
+            v = String v if v.is_a? Symbol
```

Comment:
This is the one case I'm not so sure about, since Kernel methods that begin with a capital letter otherwise look too much like constants (module names). You have cases like this, and `Array(arg)`, and some subclass constructors like `DelegateClass(type)` below.

Reply:
There's a config option for excluding methods, but I can't get it to work.

```sh
+++ b/google-cloud-logging/.rubocop.yml
@@ -20,6 +20,10 @@ Style/MethodCallWithArgsParentheses:
   Enabled: true
   EnforcedStyle: omit_parentheses
   AllowParenthesesInMultilineCall: true
+  IgnoredMethods:
+    - "Array"
+    - "DelegateClass"
+    - "String"
 Style/MethodDefParentheses:
   EnforcedStyle: require_no_parentheses
 Style/NumericLiterals:
Chriss-iMac:google-cloud-logging quartzmo$ bundle exec rake rubocop
Running RuboCop...
Inspecting 26 files
.......CC..C...C...C....C.

Offenses:

lib/google/cloud/logging/resource_descriptor/list.rb:25:35: C: Style/MethodCallWithArgsParentheses: Omit parentheses for method calls with arguments.
        class List < DelegateClass(::Array)
...

Reply:
Try dropping the double quotes

Reply:
According to the rubocop docs, `IgnoredMethods` works only for the `default` style, not for `omit_parentheses`. It was presumably added so that methods like `puts` could have parens omitted even when in general parens are required, but maybe they didn't see a use case for the converse. Maybe it would be worth opening a PR for rubocop to support it for `omit_parentheses`. It's probably just one line, checking for `ignored_method?` somewhere around [here](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/style/method_call_with_args_parentheses.rb#L154).

Reply:
@dazuma Thanks for figuring that out. I'll submit a rubocop PR. But if they don't respond in a week or two, I'll close this PR.

Reply:
Oh, I already sent https://github.com/rubocop-hq/rubocop/pull/6643

Reply:
@dazuma Great!